backend/routes/reservation.js

```diff
@@ -52,6 +85,70 @@ router.get('/all', async (req, res) => {
   res.json({ result });
 });
 
+// Update a reservation when a user needs to change a field.
+router.post('/update', async (req, res) => {
```

Comment:
On https://github.com/SE701Group3/git-brunching/pull/90
I've shown an example of putting entity identifiers in the url parameters.
Currently the enpoint url looks like 
`POST /reservation/update?reservationID=300` 
But what we want is
`PUT /reservation/300/`
This follows the best rest principles